chef_master/source/release_notes_chefdk.rst

```diff
@@ -1,59 +1,173 @@
+
+    Affected gems:
+
+    * ``knife-ec2``
+    * ``knife-google``
+    * ``knife-vsphere``
+
+    If you leverage this functionality, please wait to update ChefDK until 4.1 is released with fixes for these gems.
+
+* **Improved Chef Generate command**
+
+  The ``chef generate`` command has been updated to produce cookbooks and repositories that match Chef's best practices.
+
+  * ``chef generate repo`` now generates a Chef repository with Policyfiles by default. You can revert to the previous roles / environment behavior with the ``--roles`` flag.
+  * ``chef generate cookbook`` now generates a cookbook with a Policyfile and no Berksfile by default. You can revert to the previous behavior with the ``--berks`` flag.
+  * ``chef generate cookbook`` now includes ChefSpecs that utilize the ChefSpec 7.3+ format. This is a much simpler syntax that requires less updating of specs as older platforms are deprecated.
+  * ``chef generate cookbook`` no longer creates cookboook files with the unnecessary ``frozen_string_literal: true`` comments.
+  * ``chef generate cookbook`` no longer generates a full Workflow (Delivery) build cookbook by default. A new ``--workflow`` flag has been added to allow generating the build cookbook. This flag replaces the previously unused ``--delivery`` flag.
+  * ``chef generate cookbook`` now generates cookbooks with metadata requiring Chef 14 or later.
+  * ``chef generate cookbook --kitchen dokken`` now generates a fully working kitchen-dokken config.
+  * ``chef generate cookbook`` now generates Test Kitchen configs with the ``product_name``/``product_version`` method of specifying Chef Infra Client releases as ``require_chef_omnibus`` will be removed in the next major Test Kitchen release.
+  * ``chef generate cookbook_file`` no longer places the specified file in a "default" folder as these aren't needed in Chef Infra Client 12 and later.
+  * ``chef generate repo`` no longer outputs the full Chef Infra Client run information while generating the repository. Similar to the `cookbook` command you can view this verbose output with the ``--verbose`` flag.
+
+* **Chef InSpec 4**
+
+  Chef InSpec has been updated to 4.3.2 which includes the new InSpec AWS resource pack with **59** new AWS resources, multi-region support, and named credentials support. This release also includes support for auditing systems that use ``ed25519`` SSH keys.
+
+* **Chef Infra Client 15**
+
+  Chef Infra Client has been updated to Chef 15 with **8** new resources, target mode prototype functionality, ``ed25519`` SSH key support, and more. See the `Chef Infra Client 15 Release Notes <https://github.com/chef/chef/blob/master/RELEASE_NOTES.md#chef-infra-client-150293>`__ for more details.
```

Comment:
note: example of release notes link to update...

Reply:
yeah that probably shouldn't be master